src/test/java/EulerTest.java

```diff
@@ -0,0 +1,62 @@
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class EulerTest {
+    @Test
+    void whenUpperLimitIs10_Return23() {
+        //given
+        int upperLimit = 10;
+        //when
```

Comment:
Przed `when` i `then` lepiej dawać spacje - przynajmniej dla mnie jest to bardziej czytelne.